<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageArea">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>72</y>
      <width>551</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>No image is loaded</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose_file"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionRotate"/>
    <addaction name="actionCrop"/>
    <addaction name="actionAdjust_size"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actionZoom_2"/>
    <addaction name="actionZoom_3"/>
    <addaction name="actionFit_to_window"/>
    <addaction name="actionNormal_size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose_file"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_3"/>
   <addaction name="actionNormal_size"/>
   <addaction name="actionZoom_2"/>
   <addaction name="actionFit_to_window"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="actionReset"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionCrop"/>
   <addaction name="actionRotate"/>
   <addaction name="actionAdjust_size"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_open.png</normaloff>:/img/res/icon_open.png</iconset>
   </property>
   <property name="text">
    <string>open...</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_save.png</normaloff>:/img/res/icon_save.png</iconset>
   </property>
   <property name="text">
    <string>save...</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_close.png</normaloff>:/img/res/icon_close.png</iconset>
   </property>
   <property name="text">
    <string>close file</string>
   </property>
   <property name="toolTip">
    <string>Close file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_exit.png</normaloff>:/img/res/icon_exit.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_rotate.png</normaloff>:/img/res/icon_rotate.png</iconset>
   </property>
   <property name="text">
    <string>rotate...</string>
   </property>
   <property name="toolTip">
    <string>Rotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_crop.png</normaloff>:/img/res/icon_crop.png</iconset>
   </property>
   <property name="text">
    <string>crop</string>
   </property>
   <property name="toolTip">
    <string>Crop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_reset.png</normaloff>:/img/res/icon_reset.png</iconset>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
   <property name="toolTip">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_zoom_out.png</normaloff>:/img/res/icon_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>zoom (-)</string>
   </property>
   <property name="toolTip">
    <string>Zoom (-)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_zoom_in.png</normaloff>:/img/res/icon_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>zoom (+)</string>
   </property>
   <property name="toolTip">
    <string>Zoom (+)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_fit_to_width.png</normaloff>:/img/res/icon_fit_to_width.png</iconset>
   </property>
   <property name="text">
    <string>fit to window</string>
   </property>
   <property name="toolTip">
    <string>Fit to window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_undo.png</normaloff>:/img/res/icon_undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_redo.png</normaloff>:/img/res/icon_redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionNormal_size">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/icon_normal.png</normaloff>:/img/res/icon_normal.png</iconset>
   </property>
   <property name="text">
    <string>normal size</string>
   </property>
   <property name="toolTip">
    <string>Normal size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionAdjust_size">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/res/resize.png</normaloff>:/img/res/resize.png</iconset>
   </property>
   <property name="text">
    <string>adjust size...</string>
   </property>
   <property name="toolTip">
    <string>Adjust Size</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>save()</slot>
  <slot>open()</slot>
  <slot>zoomIn()</slot>
 </slots>
</ui>
